web/src/libs/iconfont.js

```diff
@@ -30,7 +30,7 @@
.611623 162.599849-58.348679 0-100.360453-5.446038-137.703849-14.003925-7.780226-1.555321-14.781585-7.001358-14.781585-15.559245v-56.015698c0-8.557887 7.001358-14.781585 14.780377-14.781585h1.557736c31.118491 3.889509 108.91834 7.780226 137.703849 7.780226 69.240755 0 90.246038-24.896 90.246038-70.019622 0-30.342038-14.780377-45.901283-66.128906-76.243321l-102.694641-61.460528C20.138264 426.587774 0.688302 381.464151 0.688302 327.003774c0-85.578868 49.790792-147.818264 194.497207-147.818265 51.347321 0 122.922264 7.779019 149.373585 14.003925 7.780226 1.555321 14.005132 7.001358 14.005132 14.780377v57.572227c0 7.780226-5.446038 14.003925-13.226264 14.003924h-1.556528c-51.347321-4.668377-102.694642-7.780226-153.263094-7.780226-57.572226 0-81.690566 20.227623-81.690566 55.238038 0 25.672453 13.227472 41.232906 63.795924 69.240754l93.359094 52.124981c85.578868 47.456604 108.140679 97.248604 108.14068 158.71034z m433.99849 147.040603c0 4.66717-4.66717 9.33434-10.892075 9.33434h-94.913208c-8.559094 0-17.895849-8.556679-22.563019-15.559245l-122.143396-177.381434v176.603773c0 8.557887-7.003774 16.338113-15.560453 16.338114h-71.574943c-8.557887 0-15.560453-7.780226-15.560453-16.338114V186.965736c0-8.557887 7.002566-15.559245 15.560453-15.559245h71.574943c8.557887 0 15.559245 7.001358 15.559246 15.559245v324.422038L690.645736 334.007547c5.446038-7.001358 13.226264-9.335547 17.893434-9.335547h87.135396c7.780226 0 11.669736 3.111849 11.669736 8.557887 0 2.334189-0.77766 4.66717-3.111849 7.780226L647.856302 533.171925l157.932679 214.726037c1.555321 2.332981 2.332981 3.888302 2.332981 6.223698z m434.777359-416.225811c0 1.556528 0 3.113057-0.777661 4.668377L1112.19683 796.911094c-24.117132 84.8-52.902642 122.143396-158.710339 122.143397-25.672453 0-64.572377-4.66717-82.467019-10.113208-8.557887-2.334189-16.338113-5.446038-16.338114-13.226264v-49.790793c0-8.557887 7.002566-14.003925 15.560453-14.003924h1.556529c21.784151 0.77766 66.128906 4.668377 87.911849 4.668377 33.453887 0 50.56966-9.336755 61.461736-49.792l6.223698-23.339471h-11.669736c-22.561811 0-50.56966-5.446038-67.685434-64.573585l-101.916981-356.319397c-0.77766-2.334189-0.77766-3.889509-0.777661-5.446037 0-7.780226 4.668377-12.447396 14.782793-12.447397h78.576302c7.002566 0 13.226264 7.001358 14.781585 14.003925l78.577509 316.641811c3.111849 13.226264 6.223698 17.115774 10.892075 17.115774h7.001359l84.023547-334.536453c1.555321-7.001358 7.779019-13.225057 15.559245-13.225057h79.35517c7.780226 0 14.003925 6.223698 14.003925 13.225057z" fill="#1890FF" ></path><path d="M3643.813434 155.203623c21.086189-4.692528 42.908981-7.631698 64.788528-8.777661 219.435472-11.499472 406.839547 158.192302 418.404227 378.855849 3.95834 75.522415-14.055849 150.439849-50.383698 215.553208 154.855849-55.217509 259.634717-206.250264 250.596226-378.724227-11.004377-209.968302-189.681509-371.759094-398.479698-360.815094-114.165132 5.982189-218.470642 63.932377-284.925585 153.907925z" fill="#D8D8D8" ></path></symbol></svg>',
```

Comment:
it seems that `icondashboard` is used for `metrics` menu item, so you would better add new `svg` for `server_info` instead of cover the `icondashboard`